tox.ini

```diff
@@ -19,7 +19,6 @@ setenv =
     PYTHONPATH={toxinidir}
     postgresql: TEST_DB=postgis
     sqlite: TEST_DB=sqlite
-    sqlite: USE_TZ=True
```

Comment:
Maybe we need to add a new test suit or unit test to ensure the timezone is not being considered by bakery if this settings is `False`